javascript/ql/src/semmle/javascript/frameworks/NodeJSLib.qll

```diff
@@ -1134,4 +1134,48 @@ module NodeJSLib {
       result = moduleImport().getAPropertyRead(member)
     }
   }
+
+  /**
+   * Provides predicates for working with the "node-fetch" module and its platform-specific instances as a single module.
```

Comment:
Wouldn't it be better to mention `fetch` here, instead of `node-fetch`? `fetch` is the official spec that I suspect `node-fetch` and friends are derived from...